docs/UserGuide.md

```diff
@@ -89,21 +105,18 @@ If you can type fast, TA<sup>2</sup> can get your contact and assignment managem
 
 ### Viewing help: `help`
 
-Shows a help window with a link directing you to the user guide. Alternatively this can be access by clicking the `Help` button
-that can be found in the top left corner.
+Shows a help window with a link directing you to the user guide. Alternatively, you can access this page by clicking the `Help` button in the top left corner.
 
 ![help message](images/helpMessage.png)
 
 Format: `help`
 
-Possible usage:
-* The help window can be useful if you would like to refer to the user guide for more details of the
-available features in TA<sup>2</sup>.
-
+Possible usage: 
+* The help window will direct you back to this user guide from TA<sup>2</sup> if you need details of any available features.
 
 ### Adding a student: `add`
 
-Adds a student to the contact list with the student's relevant information.
+Adds a student to your contact list with their relevant information.
```

Comment:
a student doesn't match their in grammar

Reply:
good catch